### PR TITLE
Civ 1662 fix disability bug

### PR DESCRIPTION
**Pull request by TatsyMaslova**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-1662

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/views/features/response/statementOfMeans/disability.njk [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <main class="govuk-main-wrapper" role="main">
       {% set title = "Are you disabled?" %}
       {% set id = "disability" %}
```
Comment (laarmada) [inline src/main/views/features/response/statementOfMeans/disability.njk:11]: maybe redis doent work properly....

**Review by laarmada — CHANGES_REQUESTED** [review]
need to review the redis behaviour

File: src/main/views/features/response/statementOfMeans/disability.njk (hunk 1 above)
Comment (TatsyMaslova) [reply src/main/views/features/response/statementOfMeans/disability.njk:11]: no just checked redis saves it. it saves the model not the view ;)

File: src/main/routes/features/response/statementOfMeans/severeDisabilityController.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ router.get(CITIZEN_SEVERELY_DISABLED_URL, async (req, res) => {
```
Comment (LeonardoPalmeiro) [inline src/main/routes/features/response/statementOfMeans/severeDisabilityController.ts:21]: is it worth add try and catch?

File: src/main/routes/features/response/statementOfMeans/partner/partnerSevereDisabilityController.ts [hunk 3]
```diff
@@ -31,7 +31,7 @@ router.get(CITIZEN_PARTNER_SEVERE_DISABILITY_URL.toString(), async (req, res) =>
```
Comment (LeonardoPalmeiro) [inline src/main/routes/features/response/statementOfMeans/partner/partnerSevereDisabilityController.ts:31]: is it worth add try and catch?

Comment (LeonardoPalmeiro) [inline src/main/routes/features/response/statementOfMeans/partner/partnerSevereDisabilityController.ts:31]: is it worth add try and catch?

File: src/main/routes/features/response/statementOfMeans/partner/partnerDisabilityController.ts [hunk 4]
```diff
@@ -32,7 +32,7 @@ router.get(CITIZEN_PARTNER_DISABILITY_URL, async (req, res) => {
```
Comment (LeonardoPalmeiro) [inline src/main/routes/features/response/statementOfMeans/partner/partnerDisabilityController.ts:32]: is it worth add try and catch?

Comment (LeonardoPalmeiro) [inline src/main/routes/features/response/statementOfMeans/partner/partnerDisabilityController.ts:32]: is it worth add try and catch?

**Review by LeonardoPalmeiro — CHANGES_REQUESTED** [review]

File: src/main/routes/features/response/statementOfMeans/severeDisabilityController.ts (hunk 2 above)
Comment (TatsyMaslova) [reply src/main/routes/features/response/statementOfMeans/severeDisabilityController.ts:21]: For some reason i thought it was done in refactoring disability branch but now can see it is moved to done and hence it should be in master. I will pull master again just to double check. there are a lot of things in these controllers that need fixing - for example they don't redirect to url with the claim id. but i thought for some reason it was being fixed elsewhere

File: src/main/routes/features/response/statementOfMeans/partner/partnerSevereDisabilityController.ts (hunk 3 above)
Comment (TatsyMaslova) [reply src/main/routes/features/response/statementOfMeans/partner/partnerSevereDisabilityController.ts:31]: done with super Sergio's refactor. i merged it from master :)

File: src/main/views/features/response/statementOfMeans/disability.njk (hunk 1 above)
Comment (TatsyMaslova) [reply src/main/views/features/response/statementOfMeans/disability.njk:11]: can demo it to you if you want but as discussed it is working as expected :)

**Review by LeonardoPalmeiro — APPROVED** [review]
Perfect :)

**Review by GorilaNaranja — APPROVED** [review]

**Review by laarmada — APPROVED** [review]
works fine

**Pull request merged**